src/site/content/en/blog/pwas-on-oculus-2/index.md

```diff
@@ -0,0 +1,348 @@
+| ------------ | -------- | ------------------------------------------------------------------------------------------------------------------ |
+| `name`       | yes      | The name of the PWA. Currently Oculus only supports left-to-right languages for the name.                          |
+| `display`    | yes      | Currently this value must be `"standalone"` or `"minimal-ui"`. Oculus currently does not support any other values. |
+| `short_name` | no       | A shorter version of the app name, if needed.                                                                      |
+| `scope`      | no       | This field allows developers to specify what URL or paths should be considered as part of the app.                 |
+| `start_url`  | no       | Developers can specify a starting URL to load.                                                                     |
+
+Oculus has a number of optional **proprietary** Web App Manifest fields that can be used to
+customize the PWA experience.
+
+| Name                    | Required | Description                                                                                                                                                                                                                                                                                                                                                                                                                          |
+| ----------------------- | -------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `ovr_package_name`      | no       | Sets the package name of the APK generated for the PWA. This must be in reverse domain name notation, e.g., `"com.company.app.pwa"`. If not set, developers must provide a package name to the command line tool with the (then required) parameter `--package-name`.                                                                                                                                                                |
+| `ovr_multi_tab_enabled` | no       | If `true`, this boolean field will give the PWA a tab bar similar to Oculus Browser. In multi-tab PWAs, internal links that target a new tab (`target="_new"` or `target="_blank"`) will open in new tabs within the PWA window. This differs from single-tab PWAs where such links would open to a Oculus Browser window. **This feature is currently being standardized as [Tabbed Application Mode](/tabbed-application-mode/).** |
```

Comment:
Standardized for the web generally? Why isn't this an external link?

Reply:
In this case the link goes out to https://github.com/w3c/manifest/issues/737. I'll change.